packages/core/src/lib/CopyButton/CopyButton.tsx

```diff
@@ -26,11 +32,33 @@ export const useCopyToClipboard = (): [CopiedValue, CopyFn] => {
     return [copiedText, copy];
 };
 
+const CopyButtonIcon: FC<CopyButtonIconProps> = ({
+    fill,
+    iconSize,
+    copied,
+}) =>
+    copied ? (
+        <Check
+            title="check icon"
+            titleId="copybutton check icon"
+            fill={color.mint40}
+            fontSize={iconSize}
+        />
+    ) : (
+        <Copy
+            title="copy icon"
```

Comment:
Hi @AbhinavMV, do you think we need to show the title if we are showing the tooltip here?